### PR TITLE
minor documentation fixes in Tasks.md, index.md, indexing-service.md

### PR DESCRIPTION
**Pull request by rasahner**
In misc/Tasks.md, the specification for a realtime indexing task includes this line:
      "basePersistDirectory": "\/tmp\/realtime\/basePersist"
But, realtime indexing tasks do not use this for the location of the persist directory; they all seem to use /tmp/druid/persistent.  I'm suggesting that the line be removed.

In Configuration/index.md, I fixed a sentence that had broken grammar, I hope giving it the correct meaning.

In Configuration/indexing-service.md, it shows two endpoints using COORDINATOR_IP.  These seem like they should be OVERLORD_IP.  If I use the coordinator ip I get a Not Found error.

### CODE REVIEW

File: docs/content/misc/tasks.md [hunk 1]
```diff
@@ -251,7 +251,6 @@ The indexing service can also run real-time tasks. These tasks effectively trans
       "maxRowsInMemory": 500000,
       "intermediatePersistPeriod": "PT10m",
       "windowPeriod": "PT10m",
-      "basePersistDirectory": "\/tmp\/realtime\/basePersist",
```
Comment (himanshug) [inline docs/content/misc/tasks.md]: if this setting is neglected by the code then it is a code bug really. we should fix that instead.

Comment (rasahner) [inline docs/content/misc/tasks.md]: Eric says it's a feature that basePersistDirectory is ignored for realtime tasks (as opposed to realtime nodes).

Comment (himanshug) [inline docs/content/misc/tasks.md]: @cheddar can you please add the explanation ? Is this because we want to consolidate all things related to task in the task working directory?

Comment (rasahner) [inline docs/content/misc/tasks.md]: I think it was to have druid be in control of where the persistent data went, so that there was no interference between tasks.   @pjain1, you communicated with Eric about this, maybe you can clarify.

Comment (pjain1) [inline docs/content/misc/tasks.md]: @himanshug @rasahner this setting is ignored and Eric said it does that so that it can use the task dir for the task
so that it's storage is local to the task which is inline with what both of you said.

Comment (himanshug) [inline docs/content/misc/tasks.md]: not supporting basePersistDirectory so that "multiple tasks don't interfere with each other" makes sense.

**Comment by himanshug** [comment]
LGTM

File: docs/content/misc/tasks.md (hunk 1 above)
Comment (nishantmonu51) [inline docs/content/misc/tasks.md]: can we throw an Exception stating the root cause when user sets an invalid config, instead of silently ignoring it ?

Comment (rasahner) [inline docs/content/misc/tasks.md]: maybe we can, but not part of this PR
